frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -100,8 +100,8 @@ function parsePeopleParams(peopleParams: PeopleParamType, filters: Partial<Filte
     const { action, date_from, date_to, breakdown_value, ...restParams } = peopleParams
     const params = filterClientSideParams({
         ...filters,
-        entity_id: action.id,
-        entity_type: action.type,
+        entity_id: action.id || filters?.events?.[0].id,
```

Comment:
I think you can chain a third condition here for `filters?.actions?.[0].id`. Events work now but if an action is selected the datapoint still will fall over